packages/elements/src/datetime-picker/locales.ts

```diff
@@ -21,20 +21,20 @@ import { Phrasebook } from '@refinitiv-ui/phrasebook';
 
 // match locales against date-fns
 // This will be used with resolveLocale function
-const globals = {};
+// for 5 langauge is set in Phrasebook de / en / ja / zh / zh-hant
+const globals = {
```

Comment:
Could you import this from phrasebook instead creating a new variable. I want to reduce redundant code.